src/components/App.tsx

```diff
@@ -1,26 +1,30 @@
+import { Provider } from 'react-redux';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import store from 'slices';
 import Auth from './pages/Auth';
 import Home from './pages/Home';
 import Login from './pages/Login';
 import Main from './pages/Main';
 
 const App = () => (
-  <BrowserRouter>
-    <Switch>
-      <Route exact path="/">
-        <Home />
-      </Route>
-      <Route path="/login">
-        <Login />
-      </Route>
-      <Route path="/auth">
-        <Auth />
-      </Route>
-      <Route path="/main">
-        <Main />
-      </Route>
-    </Switch>
-  </BrowserRouter>
+  <Provider store={store}>
+    <BrowserRouter>
+      <Switch>
+        <Route exact path="/">
+          <Home />
+        </Route>
```

Comment:
```suggestion
        <Route exact path="/" component={<Home />}>
```
요렇게 작성하면 라우트 되는 부분을 좀 더 명확하게 파악할 수 있지 않을까용!?

Reply:
훨 깔끔하네용! `bf213c4`에서 적용 완료하였습니다!